shell/packages/sandstorm-db/db.js

```diff
@@ -112,6 +112,7 @@ DevApps = new Mongo.Collection("devapps");
 //
 // Each contains:
 //   _id:  The application ID string (as with Packages.appId).
+//   appId: The same as _id, included to mimic Packages.appId.
```

Comment:
Maybe, though perhaps if we were being ORM-y we'd want some third entity that is the data shared between the two.  The fields that Packages have that DevApps don't have mostly relate to installation lifecycle, as far as I can tell:

* `status` would always be "ready" for devApps
* `progress` would be -1 (N/A)
* `error` is not applicable since `status` is always `ready`
* `shouldCleanup` relates to uninstalling apps, which you don't do with devApps
* `url` is only relevant when `status` is `download`, so N/A
* `isAutoUpdated` would be false, though maybe semantically we would want dev apps to not get cleaned up

That leaves:

* `authorPgpKeyFingerprint`, which we should maybe add to DevApps and attempt to populate from spk dev
* `timestamp`, which is unique to DevApps
* `packageId`, which is unique to DevApps (perhaps we should replace DevApps._id with packageId, and do `DevApps.findOne({appId: appId})` anywhere we relied on _id being an appId?)

In some ways it's like Packages and DevApps both extend some base that has an appId, a packageId, and a manifest.  Not sure how to canonically represent that in Mongo, though.